packages/api-explorer/src/JsonForm/anyOf-custom-editor.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-underscore-dangle */
+const baseCustomEditor = require('./get-custom-editor')
+const Choices = require('choices.js')
+require('choices.js/public/assets/styles/choices.min.css')
+
+module.exports = (setSwitcher) => baseCustomEditor('multiple').extend({
```

Comment:
Could we add some comments to this file? It is quite difficult to understand

Reply:
done